app/src/main/res/values/colors_services.xml

```diff
@@ -22,6 +22,17 @@
     <color name="dark_soundcloud_accent_color">#FFFFFF</color>
     <color name="dark_soundcloud_statusbar_color">#ff9100</color>
 
+    <!-- PeerTube -->
+    <color name="light_peertube_primary_color">#f97d46</color>
```

Comment:
The oragne looks a little bit washed out. As the contrast compared to the icons in dark mode is a little to low for my taste.

Here is the original:
![Screenshot_20190310-001524_NewPipe_Debug](https://user-images.githubusercontent.com/1891273/54078565-75482480-42ca-11e9-9da9-2ff66b5b1893.png)

Here is a photoshoped version with a higher saturation:
![photo_shoped](https://user-images.githubusercontent.com/1891273/54078570-8002b980-42ca-11e9-80cd-42b99cc94a25.png)